coap-core/src/main/java/com/mbed/coap/server/CoapServerBuilder.java

```diff
@@ -310,6 +314,11 @@ public CoapServerBuilderForUdp duplicateMessageDetectorCache(PutOnlyMap<CoapRequ
             return this;
         }
 
+        public CoapServerBuilderForUdp duplicateMessageDetector(DuplicationDetector detector) {
```

Comment:
Internal interface is exposed on the public builder.

Reply:
I think DuplicateDetector could be public interface. Moved it to public.